### DISCUSSION TITLE
posts/ultimate-homelab-server/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#320] [Comments]
# posts/ultimate-homelab-server/

Today I built the ultimate, all in one, HomeLab Home Server to handle everything.

https://technotim.live/posts/ultimate-homelab-server/

**Comment by hodgkiss** [comment]
Again example.com is broken. - see comment in Treafik 3 article also.

**Comment by mombe090** [comment]
Hi Tim, what is the motherboard u are using here ?
